cli/running/watchdog.go

```diff
@@ -78,19 +75,21 @@ func (wd *Watchdog) Start() error {
 	// get started for avoiding a race condition between tt start
 	// and tt stop. This way we avoid a situation when we receive
 	// a signal before starting a handler for it.
-	wd.startSignalHandling()
+	goroutineCtx, goroutineCancel := context.WithCancel(context.Background())
```

Comment:
Can we be more specific in names? All contexts are related to goroutines :) We can use `watchdogCtx` and `watchdogCancel` for example here. Or something similar that will give us a clue of context and cancel func purpose. Compare: `watchdogCancel()` and `goroutineCancel()`.

Reply:
Makes sense. I've changed it to `watchdogCtx` and `watchdogCancel`